Example-iOS/DemoStrings.swift

```diff
@@ -306,6 +306,24 @@ enum DemoStrings {
         )
     }()
 
+    // Demonstrate fractions.
+    static let fractionsExample: NSAttributedString = {
+
+        let garamondStyle = StringStyle(
+            .font(UIFont(name: "EBGaramond12-Regular", size: 24)!),
+            .lineHeightMultiple(1.2),
+            .adapt(.body)
+        )
+
+        let string = "\n1336 <fraction>6/10</fraction> + <fraction>4/10</fraction> = 1337"
```

Comment:
#req delete `\n`
#pp I think this looks better with uppercase numbers, but uppercase numbers and diagonal fractions don't mix with Garamond, so you'd have to switch back to lower in `<fraction>`:

```swift
let string = "1336 <fraction>6/10</fraction> + <fraction>4/10</fraction> = 1337"
return string.styled(with: garamondStyle.byAdding(
    .numberCase(.upper),
    .xmlRules([
        .style("fraction", garamondStyle.byAdding(.fractions(.diagonal), .color(.raizlabsRed), .numberCase(.lower))),
        ])
    )
)
```